apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/DisplayServiceAccountCard.tsx

```diff
@@ -192,64 +208,94 @@ const DisplayServiceAccountCard = (props: Props) => {
 
   return (
     <Card>
-      <Flex justifyContent="space-between" marginBottom="none">
-        <Paragraph marginBottom="none" marginTop="spacingXs">
-          <b>Google Service Account Details</b>
-        </Paragraph>
-        <Flex justifyContent="space-between" marginBottom="spacingL">
-          <Box paddingRight="spacingXs" paddingTop="spacingXs">
-            <TextLink
-              testId="editServiceAccountButton"
-              as="button"
-              variant="primary"
-              onClick={() => onInEditModeChange(true)}>
-              Edit
-            </TextLink>
-          </Box>
-          <Box style={{ minWidth: '60px', minHeight: '30px' }}>
-            {isLoadingAdminApi && isLoadingDataApi ? (
-              <Spinner variant="primary" />
-            ) : (
-              <Button variant="primary" size="small" onClick={handleApiTestClick}>
-                Test
-              </Button>
-            )}
-          </Box>
-        </Flex>
-      </Flex>
-      <FormControl>
-        <FormControl.Label marginBottom="none">Service Account</FormControl.Label>
-        <Paragraph>
-          <Flex alignItems="center">
-            <Box paddingRight="spacingS">
-              <TextLink
-                icon={<ExternalLinkTrimmedIcon />}
-                alignIcon="end"
-                href={`https://console.cloud.google.com/iam-admin/serviceaccounts/details/${serviceAccountKeyId.clientId}?project=${serviceAccountKeyId.projectId}`}
-                target="_blank"
-                rel="noopener noreferrer">
-                {serviceAccountKeyId.clientEmail}
-              </TextLink>
-            </Box>
+      {isLoadingAdminApi || isLoadingDataApi ? (
+        <Skeleton.Container>
+          <Skeleton.BodyText numberOfLines={8} />
+        </Skeleton.Container>
+      ) : (
+        <>
+          <Flex justifyContent="space-between" marginBottom="none">
+            <Paragraph marginBottom="none" marginTop="spacingXs">
+              <b>Google Service Account Details</b>
+            </Paragraph>
+            <Flex justifyContent="space-between" marginBottom="spacingL">
+              <Box paddingRight="spacingXs" paddingTop="spacingXs">
+                <TextLink
+                  testId="editServiceAccountButton"
+                  as="button"
+                  variant="primary"
+                  onClick={() => onInEditModeChange(true)}>
+                  Edit
+                </TextLink>
+              </Box>
+              <Box style={{ minWidth: '60px', minHeight: '30px' }}>
```

Comment:
nit: more inline styling here, just want to make sure we are following a pattern one way or another.